python_modules/dagster/dagster/_core/storage/asset_value_loader.py

```diff
@@ -75,11 +81,14 @@ def load_asset_value(
             python_type (Optional[Type]): The python type to load the asset as. This is what will
                 be returned inside `load_input` by `context.dagster_type.typing_type`.
             partition_key (Optional[str]): The partition of the asset to load.
+            resource_config (Optional[Any]): A dictionary of resource configurations to be passed
+                to the :py:class:`IOManager`.
```

Comment:
nit: would be more accurate to `resource configurations to be passed to resources required by the IOManager`

Reply:
it's to the IOManager too though, right?

Reply:
that is also true, yea